server/channels/api4/status.go

```diff
@@ -78,7 +78,10 @@ func getUserStatusesByIds(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write(js)
+	if _, err := w.Write(js); err != nil {
+		c.Err = model.NewAppError("getUserStatusesByIds", "api.status.write_response.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
+		return
+	}
```

Comment:
Similar comment as in the other PR: Please just log the error

Reply:
Okay okay.. Will fix all the requested changes in the PRs tomorrow... Right now I'm writing a blog, where I mentioned mattermost as one of the top repo to contribute... 🥰

Reply:
Done!